actionview/lib/action_view/helpers/number_helper.rb

```diff
@@ -26,10 +26,11 @@ def initialize(number)
 
       # Delegates to ActiveSupport::NumberHelper#number_to_phone.
       #
+      #   number_to_phone("1234567890")         # => "123-456-7890"
+      #
       # Additionally, supports a +:raise+ option that will cause
       # InvalidNumberError to be raised if +number+ is not a valid number:
       #
-      #   number_to_phone("12x34")              # => "12x34"
```

Comment:
Can you still keeps these, so we don't have to look at `ActiveSupport::NumberHelper` what happens with invalid numbers?

Reply:
Ah good point, let me restore them 👍🏻